lib/meataxe.gi

```diff
@@ -3311,21 +3311,19 @@ end;
 ##
 #F  InvariantBilinearForm( module ) . . . .
 ##
-## Look for an invariant bilinear form of the absolutely irreducible
-## GModule module. Return fail, or the matrix of the form.
+## Look for an invariant bilinear form of the GModule module.
+## Return fail, or the matrix of the form.
 SMTX.InvariantBilinearForm:=function( module  )
    local DM, iso;
 
-   if not SMTX.IsMTXModule(module) or
-                            not SMTX.IsAbsolutelyIrreducible(module) then
-      Error(
- "Argument of InvariantBilinearForm is not an absolutely irreducible module");
+   if not SMTX.IsMTXModule(module) then
+      Error("Argument of InvariantBilinearForm is not a module");
    fi;
    if IsBound(module.InvariantBilinearForm) then
      return module.InvariantBilinearForm;
    fi;
    DM:=SMTX.DualModule(module);
-   iso:=MTX.IsomorphismIrred(module,DM);
+   iso:=MTX.IsomorphismModules(module,DM);
    if iso = fail then
```

Comment:
Here, two, we might get additional problems, i.e., `iso` may not be symmetric -- we ought to add a check for that.